### ISSUE TITLE
Mapping of Geo-Coordinates in Location Set Uploads/Exports Should Be Optional

### ISSUE THREAD

**Issue by SarahCooperNDI** [nditech/apollo#682]
Having mandatory mapping of longitude and latitude for every level in the location sets makes the potential number of fields to map VERY long and greatly increases the risk that someone will accidentally map a location code to the wrong field (see below screenshot). Similarly, mandatory export of longitude and latitude in location sets makes these files very difficult to manipulate due to the high number of blank columns.  Since projects with geo-location are still fringe use cases for Apollo and not the norm could we please have a tick box when the location set is first being created **Include Latitude and Longitude** and only map and export the longitude/latitude fields when this box is ticked?

![Apollo - Longitude Latitude Mapping](https://user-images.githubusercontent.com/48285413/72825877-d4dec280-3c45-11ea-8fd3-07d5c261f069.png)

@blynchNDI @sarahmoulton @RLK-NDI

**Comment by blynchNDI** [comment]
@SarahCooperNDI very good idea - this would take care of #641 by providing a more complete solution to the overall issue

**Comment by takinbo** [comment]
A probably better suggestion would be to add the option of specifying whether GPS coordinates will be supplied for an admin division during it's creation, in the admin divisions management screen.

**Comment by blynchNDI** [comment]
True, that would allow you to enable it for some levels and not others
